features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/MediationPolicies/EditMediationPolicy.jsx

```diff
@@ -185,7 +185,11 @@ function EditMediationPolicy(props) {
             })
             .catch((errorResponse) => {
                 console.log(errorResponse);
-                Alert.error(JSON.stringify(errorResponse));
+                if (errorResponse.response.body.description !== null) {
```

Comment:
let's add the i18n support
intl.formatMessage(

Reply:
This error message is sent from back-end which can be different for different scenarios. Hence, we could not add i18n support for this.